sktime/series_as_features/base/estimators/tests/test_feature_importances_.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import pytest
+
+from sklearn.base import clone
+from sklearn.pipeline import Pipeline
+from sklearn.pipeline import FeatureUnion
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.preprocessing import FunctionTransformer
+from sktime.datasets import load_gunpoint
+from sktime.utils.time_series import time_series_slope
+from sktime.transformers.series_as_features.segment import IntervalSegmenter
+from sktime.transformers.series_as_features.compose import RowTransformer
+from sktime.transformers.series_as_features.summarize._extract import \
+    RandomIntervalFeatureExtractor
+from sktime.classification.compose._ensemble import TimeSeriesForestClassifier
+
+X_train, y_train = load_gunpoint(split="TRAIN", return_X_y=True)
+
+
+# Check results of a simple case of single estimator, single feature and
+# single interval from different but equivalent implementations
+
+
+def test_elementary_case():
+    random_state = 1234
+
+    # Compute using default method
+    features = [np.mean]
+    steps = [('transform', RandomIntervalFeatureExtractor(
+                n_intervals=1,
+                features=features,
+                random_state=random_state)),
+             ('clf', DecisionTreeClassifier())]
+    base_estimator = Pipeline(steps)
+    clf1 = TimeSeriesForestClassifier(estimator=base_estimator,
+                                      random_state=random_state,
+                                      n_estimators=1)
+    clf1.fit(X_train, y_train)
+
+    # Extract the interval and the estimator, and compute using pipelines
+    intervals = clf1.estimators_[0].steps[0][1].intervals_
+    steps = [
+        ('segment', IntervalSegmenter(intervals)),
+        ('transform', FeatureUnion([
+            ('mean', RowTransformer(
+                FunctionTransformer(func=np.mean, validate=False)))
+            ])),
+        ('clf', clone(clf1.estimators_[0].steps[-1][1]))
+    ]
+    clf2 = Pipeline(steps)
+    clf2.fit(X_train, y_train)
+
+    # Check for feature importances obtained from the estimators
+    fi1 = clf1.estimators_[0].steps[-1][1].feature_importances_
```

Comment:
instead of naming them `fi1` and `fi2`, it may be easier to understand if we name them, `actual` and `expected`, here and same for function below